services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -647,6 +647,39 @@ function epa_workflow_module_implements_alter(&$implementations, $hook) {
  */
 function epa_workflow_form_revision_overview_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   $form['node_revisions_table'][0]['operations']['#prefix'] = '<strong>Latest revision</strong><br>';
+
+  $node = $form_state->getBuildInfo()['args'][0];
+  if ($node instanceof NodeInterface && !$node->isNew()) {
+    // Loading will always return the "Current revision".
+    /** @var \Drupal\node\NodeInterface $current */
+    $current = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->load($node->id());
+
+    // If editing latest revision and has a Publish Date set.
```

Comment:
We're not in the editing context, so let's remove the reference to editing just for clarity.